src/DacpacTool/ScriptInspector.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+
+namespace MSBuild.Sdk.SqlProj.DacpacTool
+{
+    public sealed class ScriptInspector
+    {
+        private readonly List<string> _includedFiles = new List<string>();
+        public List<string> IncludedFiles
```

Comment:
We might want to change the type of this property to `IEnumerable<string>` since it is supposed to be readonly.

Reply:
Done